charts/karpenter/values.yaml

```diff
@@ -81,7 +81,7 @@ affinity:
 topologySpreadConstraints:
   - maxSkew: 1
     topologyKey: topology.kubernetes.io/zone
-    whenUnsatisfiable: ScheduleAnyway
+    whenUnsatisfiable: DoNotSchedule
```

Comment:
Just wondering, is it possible for MNG to provide nodes in the same zone? Do we need to include zone spread for nodes launched by MNG?

Reply:
The MNG spans every subnet that is specified when creating the group so it's possible a cx provides one and only one subnet. As a result, the MNG only launches into that zone.

Reply:
DoNotSchdule strikes me as the correct default here. Users have the ability to opt-in to `ScheudleAnyways`, but the vast majority of users are going to want these spread across both multiple nodes and multiple zones. You don't have guaranteed resiliency across zones today, so this seems like a solid change to make 👍🏼